### PR TITLE
Move classes

### PR DESCRIPTION
**Pull request by judy2k**
Some classes were in the wrong place. This moves them.

### CODE REVIEW

**Comment by judy2k** [comment]
This can't be merged until after #27

File: src/main/java/com/nexmo/client/voice/endpoints/SendDtmfMethod.java [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/voice/endpoints/SendDtmfMethod.java]: Looks like the copyright is in here twice now

File: src/main/java/com/nexmo/client/voice/endpoints/StartTalkMethod.java [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.nexmo.client.voice.endpoints;/*
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/voice/endpoints/StartTalkMethod.java]: Copyright also duplicated here

File: src/main/java/com/nexmo/client/voice/endpoints/StopTalkMethod.java [hunk 3]
```diff
@@ -1,4 +1,26 @@
...
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/voice/endpoints/StopTalkMethod.java]: Copyright duplicated

File: src/main/java/com/nexmo/client/voice/endpoints/TalkEndpoint.java [hunk 4]
```diff
@@ -1,4 +1,26 @@
...
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/client/voice/endpoints/TalkEndpoint.java]: Copyright duplicated

File: src/main/java/com/nexmo/client/voice/endpoints/SendDtmfMethod.java (hunk 1 above)
Comment (judy2k) [inline src/main/java/com/nexmo/client/voice/endpoints/SendDtmfMethod.java]: Yeah - the next PR fixes that ;-)

File: src/main/java/com/nexmo/client/voice/endpoints/StartTalkMethod.java (hunk 2 above)
Comment (judy2k) [inline src/main/java/com/nexmo/client/voice/endpoints/StartTalkMethod.java]: Ditto

**Pull request merged**